release-notes/6.0/preview/api-diff/preview1/.Net/6.0-preview1_System.Numerics.md

```diff
@@ -0,0 +1,37 @@
+# System.Numerics
+
+``` diff
+ namespace System.Numerics {
```

Comment:
Did we have a discussion around changing the names here? I recall something like that in the past but don't recall if it was for `System.Numerics.Vector` in particular...

(The new names are 100% better, IMO). CC. @pgovind

Reply:
Yea I like the new names here too. LGTM!